lib/shopify_import_massager.ex

```diff
@@ -60,6 +60,120 @@ defmodule ShopifyImportMassager do
     end)
   end
 
+  def massage_all(
```

Comment:
This is the way I want to approach things: a single function call that does everything with specific parameters. Next PR will have a lot of deletions stripping out the old way of doing things.